components/data-services/org.wso2.carbon.dataservices.core/pom.xml

```diff
@@ -224,6 +224,11 @@
             <groupId>org.json.wso2</groupId>
             <artifactId>json</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-dbutils</groupId>
+            <artifactId>commons-dbutils</artifactId>
+            <version>1.7</version>
```

Comment:
We need to move dependency version to the parent pom. And why we need this dependency?

Reply:
Removed.